settings.gradle

```diff
@@ -14,6 +14,7 @@ include 'fabric-api-bom'
 include 'fabric-api-catalog'
 
 include 'fabric-api-base'
+include 'fabric-api-dev'
```

Comment:
This should be set as a dev only module, and not included in the prod jar.

Reply:
The reason it's included in prod is since some developers requested that they would be able to use it outside of dev envs to be debugging in prod easier